Websites/webkit.org/demos/webxr-chess/index.html

```diff
@@ -0,0 +1,157 @@
+        <section class="demo">
+        
+            <sh-geometry-cylinder id="pointer" height="2" radius-bottom="0.001" radius-top="0.001" location="0 1 0" rotate="-90 0 0"></sh-geometry-cylinder>
+            <sh-material-basic color="red" id="red"></sh-material-basic>
+            <sh-material-shadow color="#000030" opacity="0.5" alpha-to-coverage="true" depth-write="true" id="shadowMat"></sh-material-shadow>
+            <sh-material-physical color="grey" roughness="0.4" id="screenmaterial"></sh-material-physical>
+            <sh-material-phong emissive="#888888" environment-map="#hdrs" shininess="0" reflectivity="0.7" id="panewhite"></sh-material-phong>
+            <sh-material-standard color="#222222" roughness="0.7" id="fontblack"></sh-material-standard>
+            <sh-material-phong emissive="#333333" combine="mix" environment-map="#hdrs" shininess="0" reflectivity="0.4"id="buttonmaterial"></sh-material-phong>
+            <!-- <sh-material-standard color="darkgrey" id="buttonmaterial"></sh-material-standard> -->
+            <img id="hdr" is="sh-texture-image" src="assets/bake.hdr" mapping="reflection">
+            <img id="hdrs" is="sh-texture-image" src="assets/baketiny.hdr" mapping="refraction">
+            <sh-xr-session id="xrsession"></sh-xr-session>
+        
+            <sh-renderer-webgl scene="sh-scene" id="renderer" power-preference="high-performance">
+                <sh-scene environment="#hdr" slot="scene">
+                    <sh-group rotate="0 30 0">
+                        <sh-model id="garden" src="assets/garden4k.glb" pointer-events="none"></sh-model>
+                        <sh-light-directional shadow-bias="-1e-5" shadow-radius="1.1" shadow-focus="#chess-group" shadow-map-size="1536 1536" location="-1 1 0" intensity="4"></sh-light-directional>
+                    </sh-group>
+
+
+                    <img id="leaf-tex" is="sh-texture-image" repeat="0.25 0.5" src="assets/leaf_sprite_variants_bc.png">
+                    <sh-geometry-cylinder id="leaf-geo" sweep-length="90" open="true" segments-radial="12" height="0.15" radius-bottom="0.1" radius-top="0.1" ></sh-geometry-cylinder>
+                    <sh-material-standard id="leaf-mat" roughness="0.8" metalness="0" map="#leaf-tex" transparent="true" backface-visibility="visible" alpha-to-coverage="true"></sh-material-standard>
+                    <!-- <sh-material-depth id="leaf-mat-depth" map="#leaf-tex" depth-packing="depth" alpha-hash="true" alpha-test="0.5"></sh-material-phong> -->
+                    <!-- <sh-mesh id="leafMesh" geometry="#leaf-geo" depth-material="#leaf-mat-depth" material="#leaf-mat" location="0 0.66 -0.65"></sh-mesh> -->
```

Comment:
Any reason to keep these commented out lines?

Reply:
They are mostly things which I wasn't sure if I wanted to keep around so I will remove them.